docs/user/guides/administrators.rst

```diff
@@ -12,6 +12,7 @@ have a look at our :doc:`/tutorial/index`.
 .. toctree::
    :maxdepth: 1
 
+   Add A Custom Domain <custom-domains>
```

Comment:
Why rename the toc entry here? This page isn't just for adding a custom domain, it also has other sections.

Reply:
This is an artifact - before, it was called "How to add a custom domain"

Reply:
Fixed it for now, let's figure out a better way to align titles, navigation titles and one-line descriptions.

Reply:
Good plan, I'm sure this will continue to shape up as we move pages around more.